apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.cs

```diff
@@ -71,7 +71,7 @@ public async Task ScheduledMaintenanceEvictsSessions()
             var sessionPool = new SessionPool(client, options, client.Logger);
             var acquisitionTask = sessionPool.AcquireSessionAsync(s_sampleDatabaseName, new TransactionOptions(), default);
 
-            await client.Scheduler.RunAndPause(TimeSpan.FromMinutes(1));
+            await client.Scheduler.RunAsync(TimeSpan.FromMinutes(1));
```

Comment:
super nit: In in TargettedSessionPoolTests the 1 minute ones are `RunForSecondsAsync(60)`, maybe having them consistent to one or the other? Here and elsewhere.

Reply:
Will do for anything 1 minute or less, but keep the larger ones. I'll actually change this PR to include beta02 and the usage of Wait as well, so it'll be a little while before that's ready to review. Will ping you when it is.

Reply:
OK, that sounds good.